### PR TITLE
Make scope hoisting prelude compatible with IE11

### LINKED ISSUE #6354: Latest nightly: SWC regression: Parcel require uses shorthand property names, failing in ie11.
Sorry for bad issue report, had a hard time creating a repro repo because I don't understand what triggers the wrapping of some code in my code.

The issue is on parcel `2.0.0-nightly.686+14a039f0` when building for target `ie11`.

The output code contains this:
```
if (parcelRequire == null) {
  parcelRequire = function(id) {
    if (id in $parcel$modules) {
      return $parcel$modules[id].exports;
    }
    if (id in $parcel$inits) {
      let init = $parcel$inits[id];
      delete $parcel$inits[id];
      let module = {id, exports: {}};
      $parcel$modules[id] = module;
      init.call(module.exports, module, module.exports);
      return module.exports;
    }
    var err = new Error("Cannot find module '" + id + "'");
    err.code = 'MODULE_NOT_FOUND';
    throw err;
  };

  parcelRequire.register = function register(id, init) {
    $parcel$inits[id] = init;
  };

  $parcel$global["parcelRequirea9f6"] = parcelRequire;
}
```

And this throws an error in ie11:
```
      let module = {id, exports: {}};
```
because it has to be `id: id` for ie11 to understand it.
      
@mischnic is this enough for you to understand and fix? 👀 The issue is similar to https://github.com/parcel-bundler/parcel/issues/5555

### PR DESCRIPTION
**Pull request by mischnic**
Because it's not being transpiled

Closes #6354

### CODE REVIEW

**Review by DeMoorJasper — APPROVED** [review]

**Comment by parcel-benchmark** [comment]
## Benchmark Results
<details><summary>Kitchen Sink ✅</summary><p>

#### Timings

| Description | Time | Difference |
| --- | --- | --- |
| Cold | 1.92s | +288.00ms ⚠️ |
| Cached | 423.00ms | +62.00ms ⚠️ |

#### Cold Bundles

| Bundle | Size | Difference | Time | Difference |
| --- | --- | --- | --- | --- |
| dist/legacy/parcel.d5807e82.webp | 102.94kb | +0.00b | 502.00ms | +71.00ms ⚠️ |
| dist/modern/parcel.d5807e82.webp | 102.94kb | +0.00b | 553.00ms | +79.00ms ⚠️ |
| dist/legacy/index.0aef9bfa.js | 2.00kb | +0.00b | 1.06s | +530.00ms ⚠️ |
| dist/modern/index.ed84bbeb.js | 2.00kb | +0.00b | 1.06s | +531.00ms ⚠️ |
| dist/legacy/index.html | 701.00b | +0.00b | 574.00ms | +80.00ms ⚠️ |
| dist/modern/index.html | 701.00b | +0.00b | 574.00ms | +80.00ms ⚠️ |
| dist/legacy/index.34c47dd6.css | 77.00b | +0.00b | 1.06s | +189.00ms ⚠️ |
| dist/modern/index.45951b07.css | 77.00b | +0.00b | 1.06s | +189.00ms ⚠️ |

#### Cached Bundles

| Bundle | Size | Difference | Time | Difference |
| --- | --- | --- | --- | --- |
| dist/legacy/parcel.d5807e82.webp | 102.94kb | +0.00b | 67.00ms | +18.00ms ⚠️ |
| dist/legacy/index.0aef9bfa.js | 2.00kb | +0.00b | 85.00ms | +19.00ms ⚠️ |
| dist/modern/index.ed84bbeb.js | 2.00kb | +0.00b | 85.00ms | +19.00ms ⚠️ |
| dist/legacy/index.html | 701.00b | +0.00b | 86.00ms | +19.00ms ⚠️ |
| dist/modern/index.html | 701.00b | +0.00b | 85.00ms | +19.00ms ⚠️ |
| dist/legacy/index.34c47dd6.css | 77.00b | +0.00b | 85.00ms | +19.00ms ⚠️ |
| dist/modern/index.45951b07.css | 77.00b | +0.00b | 85.00ms | +19.00ms ⚠️ |

</p></details>

<details><summary>React HackerNews ✅</summary><p>

#### Timings

| Description | Time | Difference |
| --- | --- | --- |
| Cold | 10.68s | +914.00ms ⚠️ |
| Cached | 596.00ms | -15.00ms |

#### Cold Bundles

| Bundle | Size | Difference | Time | Difference |
| --- | --- | --- | --- | --- |
| dist/PermalinkedComment.0c75a695.js | 4.19kb | +0.00b | 419.00ms | +44.00ms ⚠️ |
| dist/UserProfile.031c1548.js | 1.56kb | +0.00b | 419.00ms | +44.00ms ⚠️ |
| dist/NotFound.c3254f12.js | 417.00b | +0.00b | 419.00ms | +44.00ms ⚠️ |

#### Cached Bundles

| Bundle | Size | Difference | Time | Difference |
| --- | --- | --- | --- | --- |
| dist/index.js | 491.15kb | +0.00b | 84.00ms | +6.00ms ⚠️ |
| dist/PermalinkedComment.0c75a695.js | 4.19kb | +0.00b | 84.00ms | +5.00ms ⚠️ |
| dist/UserProfile.031c1548.js | 1.56kb | +0.00b | 84.00ms | +5.00ms ⚠️ |
| dist/NotFound.c3254f12.js | 417.00b | +0.00b | 84.00ms | +5.00ms ⚠️ |
| dist/logo.24c8bf9e.png | 274.00b | +0.00b | 53.00ms | -3.00ms 🚀 |

</p></details>

<details><summary>AtlasKit Editor ✅</summary><p>

#### Timings

| Description | Time | Difference |
| --- | --- | --- |
| Cold | 1.13m | -4.28s 🚀 |
| Cached | 2.45s | -573.00ms 🚀 |

#### Cold Bundles

| Bundle | Size | Difference | Time | Difference |
| --- | --- | --- | --- | --- |
| dist/index.61dfa705.js | 2.60mb | +12.00b ⚠️ | 52.72s | -1.85s |
| dist/pdf.worker.a3e718f4.js | 727.98kb | +0.00b | 29.87s | -1.64s 🚀 |
| dist/editorView.98ed312c.js | 594.99kb | +0.00b | 29.86s | -1.66s 🚀 |
| dist/pdfRenderer.4dcd091a.js | 411.56kb | +0.00b | 14.22s | -1.84s 🚀 |
| dist/popup.150f7b8b.js | 209.85kb | +0.00b | 14.26s | -1.84s 🚀 |
| dist/EmojiPickerComponent.80d607bb.js | 147.28kb | +0.00b | 11.60s | -2.08s 🚀 |
| dist/Toolbar.4696372a.js | 107.27kb | +0.00b | 14.23s | -1.85s 🚀 |
| dist/media-viewer.9cb568d1.js | 74.99kb | +0.00b | 7.25s | -1.22s 🚀 |
| dist/card.859c386d.js | 62.67kb | +0.00b | 7.23s | -1.23s 🚀 |
| dist/card.2e2ae103.js | 60.71kb | +0.00b | 11.60s | -2.07s 🚀 |
| dist/Modal.fc685f68.js | 45.33kb | +0.00b | 2.97s | -573.00ms 🚀 |
| dist/component.5ad2ffce.js | 37.59kb | +0.00b | 2.97s | -573.00ms 🚀 |
| dist/esm.bbc9e6e6.js | 33.24kb | +0.00b | 7.23s | -1.23s 🚀 |
| dist/component.8b3093bc.js | 24.95kb | +0.00b | 2.96s | -573.00ms 🚀 |
| dist/DatePicker.1fbfe190.js | 22.99kb | +0.00b | 7.10s | -1.32s 🚀 |
| dist/js.85def1c2.js | 17.33kb | +0.00b | 2.95s | -560.00ms 🚀 |
| dist/ui.581881e3.js | 14.94kb | +0.00b | 7.11s | -1.31s 🚀 |
| dist/smartMediaEditor.2b8dd49b.js | 13.25kb | +0.00b | 12.41s | -3.66s 🚀 |
| dist/dropzone.b02927c7.js | 12.15kb | +0.00b | 7.13s | -1.33s 🚀 |
| dist/workerHasher.95f27d4e.js | 11.81kb | +0.00b | 7.10s | -1.32s 🚀 |
| dist/component.9e1648d9.js | 6.82kb | +0.00b | 2.95s | -560.00ms 🚀 |
| dist/card.006509f3.js | 5.96kb | +0.00b | 7.12s | -1.30s 🚀 |
| dist/media-viewer.b1bdbe97.js | 4.51kb | +0.00b | 7.12s | -1.30s 🚀 |
| dist/media-viewer.9cc3198b.js | 3.98kb | +0.00b | 7.12s | -1.30s 🚀 |
| dist/EmojiPickerComponent.1fb8000b.js | 3.72kb | +0.00b | 7.11s | -1.31s 🚀 |
| dist/png-chunks-extract.26a58e9e.js | 3.58kb | +0.00b | 2.95s | -560.00ms 🚀 |
| dist/index.5fac2a57.css | 3.46kb | +0.00b | 30.31s | -1.66s 🚀 |
| dist/dropzone.65b83492.js | 3.29kb | +0.00b | 7.13s | -1.30s 🚀 |
| dist/Modal.97b7b979.js | 3.15kb | +0.00b | 2.96s | -573.00ms 🚀 |
| dist/clipboard.17831ad8.js | 2.93kb | +0.00b | 7.13s | -1.32s 🚀 |
| dist/16.ca688077.js | 2.36kb | +0.00b | 2.96s | -573.00ms 🚀 |
| dist/ResourcedEmojiComponent.425492a2.js | 2.12kb | +0.00b | 7.11s | -1.31s 🚀 |
| dist/card.1653aa06.js | 2.06kb | +0.00b | 7.12s | -1.30s 🚀 |
| dist/date.13837189.js | 1.86kb | +0.00b | 3.30s | -604.00ms 🚀 |
| dist/images.a51e31c2.js | 1.80kb | +0.00b | 3.30s | -604.00ms 🚀 |
| dist/feedback.a4084b43.js | 1.77kb | +0.00b | 3.35s | -623.00ms 🚀 |
| dist/16.b8c83009.js | 1.75kb | +0.00b | 2.96s | -549.00ms 🚀 |
| dist/browser.8fd7afde.js | 1.69kb | +0.00b | 7.13s | -1.32s 🚀 |
| dist/16.f48613ba.js | 1.68kb | +0.00b | 2.96s | -559.00ms 🚀 |
| dist/workerHasher.a91c28d5.js | 1.63kb | +0.00b | 7.11s | -1.31s 🚀 |
| dist/workerHasher.a70276d4.js | 1.63kb | +0.00b | 11.60s | -2.06s 🚀 |
| dist/list-number.494da4e7.js | 1.59kb | +0.00b | 3.32s | -583.00ms 🚀 |
| dist/status.8783408d.js | 1.59kb | +0.00b | 3.32s | -582.00ms 🚀 |
| dist/code.677bd70c.js | 1.51kb | +0.00b | 3.30s | -603.00ms 🚀 |
| dist/heading6.2df789d9.js | 1.51kb | +0.00b | 3.35s | -623.00ms 🚀 |
| dist/heading3.99e77e15.js | 1.49kb | +0.00b | 3.32s | -650.00ms 🚀 |
| dist/link.eecf8ee1.js | 1.43kb | +0.00b | 3.30s | -603.00ms 🚀 |
| dist/16.0f105e82.js | 1.40kb | +0.00b | 2.96s | -550.00ms 🚀 |
| dist/heading5.c9eff376.js | 1.38kb | +0.00b | 3.35s | -623.00ms 🚀 |
| dist/emoji.e0bb5bdd.js | 1.36kb | +0.00b | 3.30s | -604.00ms 🚀 |
| dist/16.2475b7ab.js | 1.35kb | +0.00b | 2.96s | -559.00ms 🚀 |
| dist/16.24ecf6d5.js | 1.35kb | +0.00b | 2.96s | -551.00ms 🚀 |
| dist/16.fe096c73.js | 1.33kb | +0.00b | 3.24s | -527.00ms 🚀 |
| dist/16.9c0dd14e.js | 1.33kb | +0.00b | 2.95s | -560.00ms 🚀 |
| dist/heading2.689c1725.js | 1.32kb | +0.00b | 3.32s | -650.00ms 🚀 |
| dist/16.f6f07bdf.js | 1.29kb | +0.00b | 2.96s | -550.00ms 🚀 |
| dist/expand.c97517e7.js | 1.29kb | +0.00b | 4.38s | -878.00ms 🚀 |
| dist/heading4.a48496c5.js | 1.27kb | +0.00b | 3.32s | -651.00ms 🚀 |
| dist/16.abb96752.js | 1.25kb | +0.00b | 2.96s | -549.00ms 🚀 |
| dist/16.8a916ec3.js | 1.22kb | +0.00b | 2.95s | -560.00ms 🚀 |
| dist/16.5ab0c52f.js | 1.21kb | +0.00b | 2.96s | -549.00ms 🚀 |
| dist/16.17b379b7.js | 1.20kb | +0.00b | 3.29s | -603.00ms 🚀 |
| dist/mention.7513d265.js | 1.20kb | +0.00b | 3.32s | -583.00ms 🚀 |
| dist/layout.a2215d1e.js | 1.18kb | +0.00b | 3.30s | -604.00ms 🚀 |
| dist/Modal.71b21a38.js | 1.17kb | +0.00b | 2.95s | -560.00ms 🚀 |
| dist/16.dc7fba57.js | 1.16kb | +0.00b | 2.96s | -550.00ms 🚀 |
| dist/heading1.53882705.js | 1.16kb | +0.00b | 3.32s | -582.00ms 🚀 |
| dist/16.4947d5d2.js | 1.16kb | +0.00b | 2.95s | -560.00ms 🚀 |
| dist/divider.61f72909.js | 1.16kb | +0.00b | 3.30s | -604.00ms 🚀 |
| dist/quote.ac356b63.js | 1.15kb | +0.00b | 3.32s | -582.00ms 🚀 |
| dist/16.61848cd9.js | 1.15kb | +0.00b | 2.96s | -551.00ms 🚀 |
| dist/16.f88600cd.js | 1.15kb | +0.00b | 2.96s | -550.00ms 🚀 |
| dist/16.3457832f.js | 1.15kb | +0.00b | 2.96s | -550.00ms 🚀 |
| dist/16.59da43e0.js | 1.15kb | +0.00b | 2.95s | -559.00ms 🚀 |
| dist/action.03622130.js | 1.13kb | +0.00b | 3.30s | -603.00ms 🚀 |
| dist/component.0faea072.js | 1.12kb | +0.00b | 2.96s | -573.00ms 🚀 |
| dist/media-card-analytics-error-boundary.92f81a75.js | 1.12kb | +0.00b | 7.12s | -1.30s 🚀 |
| dist/decision.e8963abf.js | 1.12kb | +0.00b | 3.30s | -604.00ms 🚀 |
| dist/panel-warning.d87689fa.js | 1.11kb | +0.00b | 3.32s | -583.00ms 🚀 |
| dist/16.71584d67.js | 1.11kb | +0.00b | 2.95s | -560.00ms 🚀 |
| dist/list.452d2378.js | 1.08kb | +0.00b | 3.32s | -583.00ms 🚀 |
| dist/16.d79d0001.js | 1.07kb | +0.00b | 2.96s | -549.00ms 🚀 |
| dist/panel-error.2d1266a1.js | 1.01kb | +0.00b | 3.32s | -583.00ms 🚀 |
| dist/panel.2fa2f198.js | 1.01kb | +0.00b | 3.32s | -583.00ms 🚀 |
| dist/table.5143dd29.js | 1022.00b | +0.00b | 3.32s | -582.00ms 🚀 |
| dist/panel-success.8c4cf740.js | 978.00b | +0.00b | 3.32s | -583.00ms 🚀 |
| dist/panel-note.aa22bfaa.js | 974.00b | +0.00b | 3.32s | -583.00ms 🚀 |
| dist/media-viewer-analytics-error-boundary.3b0c2a90.js | 964.00b | +0.00b | 7.13s | -1.30s 🚀 |
| dist/media-picker-analytics-error-boundary.da55e63c.js | 964.00b | +0.00b | 7.13s | -1.33s 🚀 |
| dist/media-card-analytics-error-boundary.0bc87153.js | 960.00b | +0.00b | 7.12s | -1.30s 🚀 |
| dist/media-card-analytics-error-boundary.fd1b1d47.js | 960.00b | +0.00b | 7.13s | -1.32s 🚀 |
| dist/simpleHasher.74d2cfec.js | 641.00b | +0.00b | 7.11s | -1.31s 🚀 |
| dist/simpleHasher.5278a07e.js | 641.00b | +0.00b | 12.41s | -3.66s 🚀 |
| dist/index.html | 119.00b | +0.00b | 2.90s | -525.00ms 🚀 |

#### Cached Bundles

| Bundle | Size | Difference | Time | Difference |
| --- | --- | --- | --- | --- |
| dist/index.780b4f4b.js | 2.60mb | -12.00b 🚀 | 167.00ms | -12.00ms 🚀 |
| dist/pdf.worker.19255c11.js | 727.98kb | +0.00b | 220.00ms | -30.00ms 🚀 |
| dist/editorView.870730b2.js | 594.99kb | +0.00b | 225.00ms | -25.00ms 🚀 |
| dist/pdfRenderer.5fa7fd49.js | 411.56kb | +0.00b | 220.00ms | -30.00ms 🚀 |
| dist/popup.67f08b14.js | 209.85kb | +0.00b | 220.00ms | -30.00ms 🚀 |
| dist/EmojiPickerComponent.e758f5eb.js | 147.28kb | +0.00b | 219.00ms | -25.00ms 🚀 |
| dist/Toolbar.3843c640.js | 107.27kb | +0.00b | 225.00ms | -25.00ms 🚀 |
| dist/media-viewer.18da36cc.js | 74.99kb | +0.00b | 220.00ms | -30.00ms 🚀 |
| dist/card.dba6de98.js | 62.67kb | +0.00b | 220.00ms | -30.00ms 🚀 |
| dist/card.21c26d44.js | 60.71kb | +0.00b | 220.00ms | -30.00ms 🚀 |
| dist/Modal.b4fb86de.js | 45.33kb | -15.00b 🚀 | 166.00ms | -16.00ms 🚀 |
| dist/component.c32f993e.js | 37.59kb | +0.00b | 166.00ms | -12.00ms 🚀 |
| dist/esm.b52ecc4d.js | 33.24kb | +0.00b | 220.00ms | -24.00ms 🚀 |
| dist/component.ba554aa1.js | 24.95kb | +0.00b | 167.00ms | -16.00ms 🚀 |
| dist/DatePicker.f8588715.js | 22.99kb | +0.00b | 219.00ms | -21.00ms 🚀 |
| dist/js.85def1c2.js | 17.33kb | +0.00b | 167.00ms | -16.00ms 🚀 |
| dist/ui.8e6dbd30.js | 14.94kb | +0.00b | 220.00ms | -30.00ms 🚀 |
| dist/smartMediaEditor.65408478.js | 13.25kb | +0.00b | 225.00ms | -25.00ms 🚀 |
| dist/dropzone.b02927c7.js | 12.15kb | +0.00b | 220.00ms | -30.00ms 🚀 |
| dist/workerHasher.95f27d4e.js | 11.81kb | +0.00b | 220.00ms | -24.00ms 🚀 |
| dist/component.9e1648d9.js | 6.82kb | +0.00b | 167.00ms | -16.00ms 🚀 |
| dist/card.006509f3.js | 5.96kb | +0.00b | 220.00ms | -30.00ms 🚀 |
| dist/media-viewer.b1bdbe97.js | 4.51kb | +0.00b | 220.00ms | -30.00ms 🚀 |
| dist/media-viewer.9cc3198b.js | 3.98kb | +0.00b | 220.00ms | -30.00ms 🚀 |
| dist/EmojiPickerComponent.1fb8000b.js | 3.72kb | +0.00b | 220.00ms | -24.00ms 🚀 |
| dist/png-chunks-extract.26a58e9e.js | 3.58kb | +0.00b | 167.00ms | -16.00ms 🚀 |
| dist/index.5fac2a57.css | 3.46kb | +0.00b | 228.00ms | -26.00ms 🚀 |
| dist/dropzone.541f8de6.js | 3.29kb | +0.00b | 220.00ms | -30.00ms 🚀 |
| dist/Modal.97b7b979.js | 3.15kb | +0.00b | 166.00ms | -16.00ms 🚀 |
| dist/clipboard.c00ab826.js | 2.93kb | +0.00b | 220.00ms | -30.00ms 🚀 |
| dist/16.ca688077.js | 2.36kb | +0.00b | 166.00ms | -12.00ms 🚀 |
| dist/ResourcedEmojiComponent.425492a2.js | 2.12kb | +0.00b | 220.00ms | -30.00ms 🚀 |
| dist/card.1653aa06.js | 2.06kb | +0.00b | 220.00ms | -30.00ms 🚀 |
| dist/date.13837189.js | 1.86kb | +0.00b | 184.00ms | -15.00ms 🚀 |
| dist/images.a51e31c2.js | 1.80kb | +0.00b | 184.00ms | -24.00ms 🚀 |
| dist/feedback.a4084b43.js | 1.77kb | +0.00b | 192.00ms | -20.00ms 🚀 |
| dist/16.b8c83009.js | 1.75kb | +0.00b | 167.00ms | -16.00ms 🚀 |
| dist/browser.8fd7afde.js | 1.69kb | +0.00b | 220.00ms | -30.00ms 🚀 |
| dist/16.f48613ba.js | 1.68kb | +0.00b | 168.00ms | -14.00ms 🚀 |
| dist/workerHasher.a91c28d5.js | 1.63kb | +0.00b | 220.00ms | -24.00ms 🚀 |
| dist/workerHasher.a70276d4.js | 1.63kb | +0.00b | 220.00ms | -30.00ms 🚀 |
| dist/list-number.494da4e7.js | 1.59kb | +0.00b | 185.00ms | -23.00ms 🚀 |
| dist/status.8783408d.js | 1.59kb | +0.00b | 187.00ms | -24.00ms 🚀 |
| dist/code.677bd70c.js | 1.51kb | +0.00b | 184.00ms | -15.00ms 🚀 |
| dist/heading6.2df789d9.js | 1.51kb | +0.00b | 192.00ms | -20.00ms 🚀 |
| dist/heading3.99e77e15.js | 1.49kb | +0.00b | 191.00ms | -20.00ms 🚀 |
| dist/link.eecf8ee1.js | 1.43kb | +0.00b | 184.00ms | -24.00ms 🚀 |
| dist/16.0f105e82.js | 1.40kb | +0.00b | 167.00ms | -16.00ms 🚀 |
| dist/heading5.c9eff376.js | 1.38kb | +0.00b | 192.00ms | -19.00ms 🚀 |
| dist/emoji.e0bb5bdd.js | 1.36kb | +0.00b | 184.00ms | -24.00ms 🚀 |
| dist/16.2475b7ab.js | 1.35kb | +0.00b | 167.00ms | -15.00ms 🚀 |
| dist/16.24ecf6d5.js | 1.35kb | +0.00b | 168.00ms | -14.00ms 🚀 |
| dist/16.9c0dd14e.js | 1.33kb | +0.00b | 166.00ms | -12.00ms 🚀 |
| dist/heading2.689c1725.js | 1.32kb | +0.00b | 191.00ms | -20.00ms 🚀 |
| dist/16.f6f07bdf.js | 1.29kb | +0.00b | 168.00ms | -15.00ms 🚀 |
| dist/expand.c97517e7.js | 1.29kb | +0.00b | 192.00ms | -20.00ms 🚀 |
| dist/heading4.a48496c5.js | 1.27kb | +0.00b | 192.00ms | -19.00ms 🚀 |
| dist/16.abb96752.js | 1.25kb | +0.00b | 167.00ms | -16.00ms 🚀 |
| dist/16.8a916ec3.js | 1.22kb | +0.00b | 166.00ms | -16.00ms 🚀 |
| dist/16.5ab0c52f.js | 1.21kb | +0.00b | 167.00ms | -16.00ms 🚀 |
| dist/16.17b379b7.js | 1.20kb | +0.00b | 184.00ms | -15.00ms 🚀 |
| dist/mention.7513d265.js | 1.20kb | +0.00b | 186.00ms | -24.00ms 🚀 |
| dist/layout.a2215d1e.js | 1.18kb | +0.00b | 184.00ms | -24.00ms 🚀 |
| dist/Modal.71b21a38.js | 1.17kb | +0.00b | 167.00ms | -15.00ms 🚀 |
| dist/16.dc7fba57.js | 1.16kb | +0.00b | 167.00ms | -16.00ms 🚀 |
| dist/heading1.53882705.js | 1.16kb | +0.00b | 191.00ms | -20.00ms 🚀 |
| dist/16.4947d5d2.js | 1.16kb | +0.00b | 166.00ms | -12.00ms 🚀 |
| dist/divider.61f72909.js | 1.16kb | +0.00b | 184.00ms | -15.00ms 🚀 |
| dist/quote.ac356b63.js | 1.15kb | +0.00b | 187.00ms | -24.00ms 🚀 |
| dist/16.61848cd9.js | 1.15kb | +0.00b | 168.00ms | -14.00ms 🚀 |
| dist/16.f88600cd.js | 1.15kb | +0.00b | 168.00ms | -15.00ms 🚀 |
| dist/16.3457832f.js | 1.15kb | +0.00b | 167.00ms | -16.00ms 🚀 |
| dist/16.59da43e0.js | 1.15kb | +0.00b | 167.00ms | -15.00ms 🚀 |
| dist/action.03622130.js | 1.13kb | +0.00b | 184.00ms | -15.00ms 🚀 |
| dist/component.0faea072.js | 1.12kb | +0.00b | 166.00ms | -12.00ms 🚀 |
| dist/media-card-analytics-error-boundary.92f81a75.js | 1.12kb | +0.00b | 220.00ms | -30.00ms 🚀 |
| dist/decision.e8963abf.js | 1.12kb | +0.00b | 184.00ms | -15.00ms 🚀 |
| dist/panel-warning.d87689fa.js | 1.11kb | +0.00b | 186.00ms | -25.00ms 🚀 |
| dist/16.71584d67.js | 1.11kb | +0.00b | 167.00ms | -31.00ms 🚀 |
| dist/list.452d2378.js | 1.08kb | +0.00b | 185.00ms | -23.00ms 🚀 |
| dist/16.d79d0001.js | 1.07kb | +0.00b | 167.00ms | -16.00ms 🚀 |
| dist/panel-error.2d1266a1.js | 1.01kb | +0.00b | 186.00ms | -24.00ms 🚀 |
| dist/panel.2fa2f198.js | 1.01kb | +0.00b | 187.00ms | -24.00ms 🚀 |
| dist/table.5143dd29.js | 1022.00b | +0.00b | 191.00ms | -20.00ms 🚀 |
| dist/panel-success.8c4cf740.js | 978.00b | +0.00b | 186.00ms | -24.00ms 🚀 |
| dist/panel-note.aa22bfaa.js | 974.00b | +0.00b | 186.00ms | -24.00ms 🚀 |
| dist/media-viewer-analytics-error-boundary.c51589b7.js | 964.00b | +0.00b | 220.00ms | -30.00ms 🚀 |
| dist/media-picker-analytics-error-boundary.da55e63c.js | 964.00b | +0.00b | 220.00ms | -30.00ms 🚀 |
| dist/media-card-analytics-error-boundary.b94e1438.js | 960.00b | +0.00b | 220.00ms | -30.00ms 🚀 |
| dist/media-card-analytics-error-boundary.a9392a91.js | 960.00b | +0.00b | 220.00ms | -30.00ms 🚀 |
| dist/simpleHasher.74d2cfec.js | 641.00b | +0.00b | 220.00ms | -24.00ms 🚀 |
| dist/simpleHasher.5278a07e.js | 641.00b | +0.00b | 224.00ms | -26.00ms 🚀 |
| dist/index.html | 119.00b | +0.00b | 167.00ms | +60.00ms ⚠️ |

</p></details>

<details><summary>Three.js ✅</summary><p>

#### Timings

| Description | Time | Difference |
| --- | --- | --- |
| Cold | 7.09s | +78.00ms |
| Cached | 519.00ms | +11.00ms |

#### Cold Bundles

*No bundle changes detected.*

#### Cached Bundles

| Bundle | Size | Difference | Time | Difference |
| --- | --- | --- | --- | --- |
| dist/Three.js | 579.69kb | +0.00b | 75.00ms | +9.00ms ⚠️ |

</p></details>

**[Click here to view a detailed benchmark overview.](https://benchmark.parceljs.org/benchmark/299555487012094472)**

**Comment by devongovett** [comment]
IE11 is dead in a few months. Wonder when we can drop it... https://www.theverge.com/2021/5/19/22443997/microsoft-internet-explorer-end-of-support-date

**Comment by devongovett** [comment]
IE11 is dead in a few months. Wonder when we can drop it... https://www.theverge.com/2021/5/19/22443997/microsoft-internet-explorer-end-of-support-date

File: packages/packagers/js/src/helpers.js [hunk 1]
```diff
@@ -13,7 +13,7 @@ if (parcelRequire == null) {
     if (id in $parcel$inits) {
       let init = $parcel$inits[id];
       delete $parcel$inits[id];
-      let module = {id, exports: {}};
+      let module = {id: id, exports: {}};
```
Comment (wbinnssmith) [inline packages/packagers/js/src/helpers.js:16]: isn't the `let` declaration itself also a problem?

Comment (mischnic) [reply packages/packagers/js/src/helpers.js:16]: It seems IE11 supports some variants of `let`: https://caniuse.com/let

**Pull request merged**